scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -746,13 +919,13 @@ Function ValidateTestParameters
 
     If ($TestName -eq "TransparentGateway")
     {
-        $validatingItems += $DeviceCACertificatePath
-        $validatingItems += $DeviceCAPrimaryKeyPath
-        $validatingItems += $TrustedCACertificatePath
+        $validatingItems += $EdgeCertGenScriptDir
+        $validatingItems += $EdgeE2ERootCACertRSAFile
+        $validatingItems += $EdgeE2ERootCAKeyRSAFile
     }
 
     $validatingItems | ForEach-Object {
-        If (-Not (Test-Path $_))
+        If (-Not (Test-Path -Path $_))
```

Comment:
Does -Path work for directory as well?

Reply:
Yes. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/test-path?view=powershell-6